scripts/barclamp_mgmt_lib.rb

```diff
@@ -260,6 +260,8 @@ def generate_assets_manifest
 
   manifests = Pathname.new(CROWBAR_PATH).join("barclamps", "manifests")
 
+  manifests.mkdir unless manifests.exist?
```

Comment:
I have set up a development environment today (after not using it for ~ a month), and I haven't run into this error somehow.... when should this occur?